pkg/component/observability/monitoring/prometheus/garden/prometheusrules.go

```diff
@@ -196,6 +196,18 @@ func gardenPrometheusRule(isGardenerDiscoveryServerEnabled bool) *monitoringv1.P
 					"last hour. The pod is running on the garden cluster.",
 			},
 		},
+		{
+			Alert:  "RuntimeCACertificateAboutToExpire",
+			Expr:   intstr.FromString("probe_ssl_earliest_cert_expiry - time() < 86400 * 30"),
+			For:    ptr.To(monitoringv1.Duration("10m")),
```

Comment:
Maybe add a comment that this is supposed to be 30 days or break it down such as `24 * 3600 * 30`.

The for is optional and I think there is no need to wait 10 minutes for the alert to stay up before sending it. If a certificate is going to expire within X time then it's still going to expire 10 minutes later.